### PR TITLE
Add docs for using hostPath plugin from restricted scc

### PR DESCRIPTION
**Pull request by pmorie**
@adellape @pweil- 

Went through this UC with a customer the other day.

### CODE REVIEW

File: admin_guide/manage_scc.adoc [hunk 1]
```diff
@@ -278,3 +278,20 @@ containers to run as root:
 . Change `*runAsUser.Type*` to *MustRunAsNonRoot*.
 
 . Save the changes.
+
+[[use-the-hostpath-volume-plugin]]
+
+=== Use the hostPath volume plugin
```
Comment (adellape) [inline admin_guide/manage_scc.adoc]: Style guide stuff:

s/volume plugin/Volume Plug-in/

File: admin_guide/manage_scc.adoc [hunk 2]
```diff
@@ -278,3 +278,20 @@ containers to run as root:
...
+
+To relax the security in your cluster so that pods are allowed to use the `hostPath`
+volume plugin without granting everyone access to the *privileged* SCC:
```
Comment (adellape) [inline admin_guide/manage_scc.adoc]: s/plugin/plug-in/

File: admin_guide/manage_scc.adoc [hunk 3]
```diff
@@ -278,3 +278,20 @@ containers to run as root:
...
+
+. Edit the *restricted* SCC:
++
+----
+$ oc edit scc restricted
+----
+
+. Add `*allowHostDirVolumePlugin: true*`.
+
+. Save the changes
```
Comment (adellape) [inline admin_guide/manage_scc.adoc]: Period on end.

**Comment by adellape** [comment]
:+1:

**Comment by pweil-** [comment]
:+1:

**Comment by pmorie** [comment]
@adellape Updated

**Comment by adellape** [comment]
Thanks! Will merge as soon as the build passes.

**Pull request merged**